mathjax3-ts/output/chtml/Wrappers/msqrt.ts

```diff
@@ -153,6 +157,9 @@ export class CHTMLmsqrt extends CHTMLWrapper {
         this.addRoot(ROOT, root, sbox);
         surd.toCHTML(SURD);
         base.toCHTML(BASE);
+        if (surd.size < 0) {
```

Comment:
Can you put a comment here that contains the gist of the explanation for the PR, for future reference?

Reply:
Done.